smart-mon.spec

```diff
@@ -40,9 +42,12 @@ binary and corresponding service.
 %install
 install -m 755 -d %{buildroot}%{bin_dir}/
 install -m 755 -d %{buildroot}%{script_dir}/
+install -m 755 -d %{buildroot}%{service_dir}/
 install -m 755 -d %{buildroot}%{etc_ceph_dir}/
+install -m 755 -d %{buildroot}%{node_exporter_dir}/
 install -m 755 smartmon.sh %{buildroot}%{bin_dir}
 install -m 755 script.sh %{buildroot}%{script_dir}
+install -m 644 smart.service  %{buildroot}%{service_dir}
 install -m 755 node-exporter.yml %{buildroot}%{etc_ceph_dir}
 
 %post
```

Comment:
Should remove this script, and just `enable` the service in the storage-ceph build. https://github.com/Cray-HPE/metal-provision/blob/main/roles/node_images_storage_ceph/vars/services/suse.yml#L25